seminare/organizer/forms.py

```diff
@@ -33,8 +33,23 @@ class GradingForm(forms.Form):
 class ProblemForm(forms.ModelForm):
     class Meta:
         model = Problem
-        fields = ["name", "number"]
-        labels = {"name": "Názov", "number": "Číslo"}
+        fields = [
+            "name",
+            "number",
+            "file_points",
+            "judge_points",
+            "text_points",
+            "judge_namespace",
+            "judge_task",
+            "text_answer",
+        ]
+        labels = {
+            "name": "Názov",
+            "number": "Číslo",
+            "file_points": "Body za popis",
+            "judge_points": "Body za program",
+            "text_points": "Body za odpoveď",
```

Comment:
3↑ predpokladám, že sú max. body?

Reply:
ano, su to (zatial) max body, ale ten RuleEngine to bude asi brat a vypocita z toho max body